src/pages/login/index.tsx

```diff
@@ -1,41 +1,53 @@
 import React, { useCallback, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import { Center } from '~/components/Box';
 import { getDashboardPath } from '~/constants/paths';
 import { useSelector } from '~/store/hooks';
 import { login } from '~/store/slices/CreateUserSlice';
+import { LoginLayout } from './layout';
+import { LoginForm, formData } from './components/LoginForm';
+import styled from 'styled-components';
+import { Center } from '~/components/Box';
+import Logo from '~/components/Logo';
+import { COLORS } from '~/styles/variables';
+import { defaultTheme } from '~/styles/theme';
 
-export const LoginPage: React.FC = () => {
-  const { isLoading, isAuthenticated, isError } = useSelector(state => state.user);
+const Page: React.FC = () => {
```

Comment:
What was wrong with `LoginPage`  😄? Now if we want to import it somewhere, the autoimports completers will use `Page` name which is not perfect in this case